opencog/nlp/relex2logic/rule-helpers.scm

```diff
@@ -466,26 +466,26 @@
 	(define adv (cog-name adv-node))
 	(define adv_instance (cog-name adv-instance-node))
 	(ListLink
-		(InheritanceLink  (ConceptNode adv_instance) (ConceptNode adv))
-			(ImplicationLink  (PredicateNode instance) (PredicateNode verb))
-			(r2l-wordinst-Predicate instance)
-			(r2l-wordinst-concept adv_instance)
-			(InheritanceLink  (SatisfyingSetLink (PredicateNode instance)) (ConceptNode adv_instance))
+		(ImplicationLink  (PredicateNode adv_instance) (PredicateNode adv))
+		(InheritanceLink  (ConceptNode instance) (ConceptNode verb))
```

Comment:
After glancing at the wiki, there does not appear to be any clear, crisp statement as to when some linguistic structure is mapped to "predicate" and when its mapped to "concept".  The wiki should try to explain this.

More generally, I am made somewhat nervous, at the gut-intuition level, that such a distinction could ever be correct.  Lignustics is complicated, for a reason; that complexity is ignored at one's peril.